moredata/utils/osm_downloader.py

```diff
@@ -53,10 +53,10 @@ def get_places(self, place_name, key, value, query=None, tags=("name","geom")):
             properties = data["properties"]
             if "tags" in properties.keys():
                 tags = properties["tags"]
-
                 for item in tags.items():
                     key, value = item[0], item[1]
```

Comment:
I think here to simplify and don't commit ambiguity, it's better to assign these values to variables named `k` and `v`.